.travis.yml

```diff
@@ -1,19 +1,25 @@
 ---
 language: node_js
 node_js:
-  - "4"
+  - 6
+  - 8
+sudo: required
+dist: trusty
 
-sudo: false
+addons:
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
 
 cache:
-  yarn: true
-  directories:
-    - node_modules
+  yarn: false
 
 env:
   # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
-  - PERCY_ENABLE=0 EMBER_TRY_SCENARIO=ember-1.13
-  - PERCY_ENABLE=0 EMBER_TRY_SCENARIO=ember-lts-2.4
+  - PERCY_ENABLE=0 EMBER_TRY_SCENARIO=ember-lts-2.8
+  - PERCY_ENABLE=0 EMBER_TRY_SCENARIO=ember-lts-2.12
```

Comment:
Will we have to break backwards-compatibility with Ember 1.13 as part of this change? If not, let's keep the ember-1.13 try here.

Overall, if it required to drop 1.13 support, we're probably ready for that and fine with it, just need to bump a major release of this after it is merged.

Reply:
I'll look at getting Node 4 working on Travis first, and then try adding this back in.

These changes *should* work fine with Ember 1.13. I removed that version to use more recent versions of Ember and because there seemed to be some weird caching/installation behaviour with `ember-try` on Travis.